src/Google.Datastore.V1Beta3/ArrayValuePartial.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Google.Type;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Google.Datastore.V1Beta3
+{
+    // Additional helper members.
```

Comment:
No can do - the class already has XML documentation in the generated code. This is really an implementation comment.